webpack.config.js

```diff
@@ -96,15 +101,15 @@ const createClientConfig = (_env, mode) => {
     entry: "./src/client/index.tsx",
     output: {
       filename: "js/client.js",
-      publicPath: "/static/",
+      publicPath: `/static/${commitHash}`
```

Comment:
```suggestion
      publicPath: `/static/${commitHash}/`
```

Without the trailing slash the paths break:

```
Unhandled Promise Rejection: ChunkLoadError: Loading chunk 76292 failed.
(error: http://localhost:1234/static/3caf913614868f70475430fca3dcc40c61d613b5js/76292.client.js)
```